templates/1_set_up_toffy.ipynb

```diff
@@ -5,28 +5,51 @@
    "id": "ec35cb18-c771-48e6-859f-e10e074545d6",
```

Comment:
<p>we could also pull an ark-analysis here and have an update script to automatically do this + manage a git-tracked templates vs untracked scripts folder.  maybe a PR for another time</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>